### ISSUE TITLE
Nice to Have - Paginate Tables

### ISSUE THREAD

**Issue by RedBlueCarrots** [RedBlueCarrots/WikiPath#70]
Basically, in the future, we may have to accomodate dozens+ of users and challenges, and so tables may become overly long, and webpages overly long too. We should make tables paginated (pages with 10-20 items/page) to accommodate large numbers of users.

Will apply to the home page, and the (to be created) leaderboard page

**Comment by RedBlueCarrots** [comment]
going to put this on hold temporarily - will be better to work on once #54 is done
